nilearn/plotting/img_plotting.py

```diff
@@ -586,7 +611,15 @@ def plot_stat_map(stat_map_img, bg_img=MNI152TEMPLATE, cut_coords=None,
                     np.logical_not(stat_map_data._mask)])
         stat_map_max = np.nanmax(stat_map_data)
         stat_map_min = np.nanmin(stat_map_data)
+
+    if symmetric_cbar == 'auto':
+        symmetric_cbar = stat_map_min < 0
```

Comment:
i would, too. but is this expected to happen for any type of data?

On Tuesday, January 6, 2015, Alexandre Abraham <notifications@github.com>
wrote:

> In nilearn/plotting/img_plotting.py
> <https://github.com/nilearn/nilearn/pull/293#discussion-diff-22557878>:
>
> > @@ -586,7 +611,15 @@ def plot_stat_map(stat_map_img, bg_img=MNI152TEMPLATE, cut_coords=None,
> >                      np.logical_not(stat_map_data._mask)])
> >          stat_map_max = np.nanmax(stat_map_data)
> >          stat_map_min = np.nanmin(stat_map_data)
> > +
> > +    if symmetric_cbar == 'auto':
> > +        symmetric_cbar = stat_map_min < 0
>
> I would expect the symmetric colorbar to work both ways: upper part of the
> original cbar if all values are positive and lower part of the cbar if all
> values are negative.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/293/files#r22557878>.
>